docs/concepts/personal-access-token.mdx

```diff
@@ -1,36 +1,24 @@
 ---
 id: personal-access-token
-title: Personal access tokens
+title: Authorization and API Keys
+sidebar_label: Authorization
 ---
 
-A Personal Access Token (PAT) is a means of authentication you can use when directly calling the Ory APIs. The PAT is bound to the
-project where it was created. A PAT created for an administrator in project `A` won't be valid in project `B` in any capacity.
-
-For example, when calling Ory administrative APIs to [list](../cli/ory-list-identities.md), [update](../cli/ory-get-identity.md),
-or [delete](../cli/ory-delete-identity.md) identities, you must authenticate as an administrator. You can do that by passing a PAT
-in the request's `Authorization` header. The PAT must be generated for a user with administrative rights in the project in which
-you want to manage identities.
+Each project has API keys used to perform privileged operations, for example deleting an identity. To create an API Key head over
+to the Ory Console [API Key page](https://console.ory.sh/projects/current/connect).
 
 :::tip
 
-You can use PATs to call APIs from the command line or through scripts. Ory PATs are always prefixed with `ory_pat_`, which makes
-it easy to identify them when analyzing code.
+You can use API keys to call APIs from the command line or through scripts. Ory API Keys are always prefixed with `ory_apikey_`
+(or `ory_pat_`), which makes it easy to identify them when analyzing code.
 
 :::
 
-## Creating a Personal Access Token
-
-Follow these steps to create a Personal Access Token (PAT):
-
-1. Open the [Ory Console](https://console.ory.sh/) and sign in.
-2. Select **Connect** from the left navigation bar.
-3. Scroll to the bottom of the page to the **Personal Access Token** section.
-4. Click the **+** icon.
-5. Define the name of the generated token. This name is used for identification purposes only.
-6. Copy the created PAT and store it in a safe place.
-
-   :::warning
-
-   The PAT is displayed only once. If you lose the token, you must generate a new one.
+Passing the API Key is done via the `Authorization` header:
```

Comment:
Is it `API Key` or `API key`? 
In the admonition there are two different capitalization approaches used in a single line.

Reply:
What do you think should it be? I'm fine either way

Reply:
As mentioned in the [previous comment](https://github.com/ory/docs/pull/1091/files/8dd94f6b99a7ec96dcd37b2d5ea731c0687517c8#r1033460313), please use `API keys`.

Reply:
Done!